src/utils/capitalize.ts

```diff
@@ -0,0 +1,3 @@
+export default function capitalize(text: string) {
+	return `${text[0].toUpperCase()}${text.slice(1, text.length)}`;
```

Comment:
I guess we can replace .slice with .substr
${text.slice(1, text.length)} --> &{text.substr(1)}